src/ansys/geometry/core/designer/designpoint.py

```diff
@@ -22,8 +22,10 @@
 """Module for creating and managing design points."""
 
 from beartype.typing import TYPE_CHECKING, Union
+import pyvista as pv
 
 from ansys.geometry.core.math.point import Point3D
+from ansys.geometry.core.misc import UNITS
```

Comment:
Search for the actual ``UNITS`` import, not the easy access. This is critical for the documentation to render properly.